fix CMake error "An ELSEIF command was found after an ELSE command."
author: Kenneth Hoste (HPC-UGent)
--- STIR-stir_rel_3_00/src/display/CMakeLists.txt.orig	2018-05-18 18:42:00.493707771 +0200
+++ STIR-stir_rel_3_00/src/display/CMakeLists.txt	2018-05-18 18:42:19.464021925 +0200
@@ -29,7 +29,7 @@
 
 elseif("${GRAPHICS}" STREQUAL "PGM")
 	ADD_DEFINITIONS(-DSTIR_PGM)
-else()
+#else()
 # TODO
 #ifeq "$(GRAPHICS)" "MATHLINK"
 # this presumably needs a Mathlink library, depends on your system though
